scripts/benchmark_stats.py

```diff
@@ -12,6 +11,8 @@ def __init__(self, folder_paths: List[str]):
 
         self.get_requests: List[int] = []
         self.post_requests: List[int] = []
+        self.total_get_requests: int = 0
+        self.total_post_requests: int = 0
         self.total_requests: int = 0
```

Comment:
This doesn't need to be in the init anymore, it can be a property of the class and use the total_get_requests and total_post_requests to calculate it:

```python
@property
def total_requests(self):
    return self.total_get_requests + self.total_post_requests
```

Reply:
Same comment as above for Mebin, we oddly exclude thank you and session, so if I did that the numbers are wrong. This PR was a quick fix to get the request count in with a card in tech to work out what this repo is really meant to do, as its a bit of a pickle at the moment